lib/api/src/tests/versions.test.js

```diff
@@ -232,74 +158,20 @@ describe('versions API', () => {
     const { api, init, state: initialState } = initVersions({ store });
     store.setState(initialState);
 
-    fetch.mockResolvedValueOnce(newResponse);
     await init({ api: { ...api, addNotification: jest.fn() } });
 
     expect(api.getLatestVersion()).toMatchObject({
-      version: '4.0.0',
+      version: '5.2.3',
     });
   });
 
-  describe('versionUpdateAvailable', () => {
-    describe('stable current version', () => {
-      it('new latest version', async () => {
```

Comment:
There should be tests for these cases. Perhaps these need to be refactored to match the new structure?

Specifically the behavior is different depending on whether the user is running a prerelease or a stable version of storybook.

Reply:
I changed these, into a different section in the testfile, should be good now